subprojects/provider/src/test/kotlin/org/gradle/kotlin/dsl/accessors/KotlinDslAccessorsSnapshotTaskIntegrationTest.kt

```diff
@@ -39,189 +39,186 @@ class KotlinDslAccessorsSnapshotTaskIntegrationTest : AbstractIntegrationTest()
-            "testRuntime", "testRuntimeClasspath", "testRuntimeOnly")).sorted())
+val javaProjectSchema: ProjectSchema<String> = listOf(
+    "annotationProcessor",
+    "apiElements",
+    "compile", "compileClasspath", "compileOnly",
+    "implementation",
+    "runtime", "runtimeClasspath", "runtimeElements", "runtimeOnly",
+    "testAnnotationProcessor",
+    "testCompile", "testCompileClasspath", "testCompileOnly",
+    "testImplementation",
+    "testRuntime", "testRuntimeClasspath", "testRuntimeOnly"
+).let { configurationNames ->
+
+    baseProjectSchema + ProjectSchema(
+        extensions = listOf(
+            ProjectSchemaEntry("org.gradle.api.Project", "reporting", "org.gradle.api.reporting.ReportingExtension"),
+            ProjectSchemaEntry("org.gradle.api.Project", "sourceSets", "org.gradle.api.tasks.SourceSetContainer"),
+            ProjectSchemaEntry("org.gradle.api.Project", "java", "org.gradle.api.plugins.JavaPluginExtension"),
+            ProjectSchemaEntry("org.gradle.api.plugins.JavaPluginExtension", "ext", "org.gradle.api.plugins.ExtraPropertiesExtension"),
+            ProjectSchemaEntry("org.gradle.api.reporting.ReportingExtension", "ext", "org.gradle.api.plugins.ExtraPropertiesExtension"),
+            ProjectSchemaEntry("org.gradle.api.tasks.SourceSetContainer", "ext", "org.gradle.api.plugins.ExtraPropertiesExtension"),
+            ProjectSchemaEntry("org.gradle.api.tasks.SourceSet", "ext", "org.gradle.api.plugins.ExtraPropertiesExtension")
+        ),
+        conventions = listOf(
+            ProjectSchemaEntry("org.gradle.api.Project", "java", "org.gradle.api.plugins.JavaPluginConvention")
+        ),
+        containerElements = configurationNames.map(::existingConfigurationContainerElement),
+        configurations = configurationNames
+    )
+}
 
 
 private
-val groovyProjectSchema: ProjectSchema<String> =
-    ProjectSchema(
-        extensions = javaExtensionsWith(
-            listOf(
-                ProjectSchemaEntry(
-                    "org.gradle.api.Project",
-                    "groovyRuntime",
-                    "org.gradle.api.tasks.GroovyRuntime"),
-                ProjectSchemaEntry(
-                    "org.gradle.api.tasks.GroovyRuntime",
-                    "ext",
-                    "org.gradle.api.plugins.ExtraPropertiesExtension"))),
-        conventions = javaProjectSchema.conventions,
-        configurations = javaProjectSchema.configurations)
+val groovyProjectSchema: ProjectSchema<String> = javaProjectSchema + ProjectSchema(
+    extensions = listOf(
+        ProjectSchemaEntry("org.gradle.api.Project", "groovyRuntime", "org.gradle.api.tasks.GroovyRuntime"),
+        ProjectSchemaEntry("org.gradle.api.tasks.GroovyRuntime", "ext", "org.gradle.api.plugins.ExtraPropertiesExtension")
+    ),
+    conventions = emptyList(),
+    containerElements = emptyList(),
+    configurations = emptyList()
+)
 
 
 private
-val kotlinDslProjectSchema: ProjectSchema<String> =
```

Comment:
⭕️ this will conflict with the Kotlin 1.3. changes :(

Reply:
Merged `develop` and fixed conflicts